packages/botfuel-dialog/src/adapters/web-adapter.js

```diff
@@ -56,6 +64,25 @@ class WebAdapter extends Adapter {
     throw new MissingImplementationError();
   }
 
+  /**
+   * Webhook used for handling template requests.
+   * @async
+   * @param {Object} req - the request object
+   * @param {Object} res - the response object
+   * @returns {Promise.<void>}
+   */
+  async handleTemplate(req, res) {
+    logger.info('handleTemplate');
+    res.setHeader('Content-Type', 'text/html; charset=utf-8');
+    fs.readFile(`src/templates/${req.params.id}.handlebars`, 'utf8', (err, data) => {
+      if (err) {
+        throw err;
+      } else {
+        res.end(Handlebars.compile(data)(req.query));
```

Comment:
We don't catch Handlebars compilation exceptions, I would have add a try/catch block to handle them and adapt the response with something like:

```javascript
try {
  const html = Handlebars.compile(data)(req.query);
  res.status(200).send(html);
} catch (e) {
  res.status(400).send(e);
}
```